content/docs/sidebar.json

```diff
@@ -525,5 +525,33 @@
         "slug": "dvclive-with-dvc"
       }
     ]
+  },
+  {
+    "label": "Studio",
```

Comment:
Let's put it before DVCLive?

![image](https://user-images.githubusercontent.com/1477535/117728717-695c9380-b1af-11eb-82bd-510f1ff0c656.png)